spec/locale.html

```diff
@@ -12,7 +12,6 @@ <h1>The Intl.Locale Constructor</h1>
       <h1>Intl.Locale ( _tag_ [ , _options_ ] )</h1>
 
       <p>
-        The following algorithm refers to the `type` nonterminal from <a href="https://www.unicode.org/reports/tr35/#Unicode_locale_identifier">UTS 35's Unicode Locale Identifier grammar</a>.
```

Comment:
why do you want to remove this references?

Reply:
Because it has now been consolidated into the section to which the "language tag nonterminal" phrase in the algorithm automatically links.